.github/scripts/generate_binary_build_matrix.py

```diff
@@ -325,6 +325,7 @@ def generate_wheels_matrix(
     os: str,
     arches: Optional[List[str]] = None,
     python_versions: Optional[List[str]] = None,
+    use_split_build: Optional[bool] = False,
```

Comment:
Why is it optional? How `use_split_build is None` are semantically different than `use_split_build = False`?
```suggestion
    use_split_build: bool = False,
```